cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -122,6 +122,10 @@ msgstr "Enviar una queja"
 msgid "Search"
 msgstr "Buscar"
 
+#: jinja2/v1/_includes/molecules/global-search.html:26
+msgid "Search the consumerfinance.gov website"
+msgstr "Busque en el sitio web consumerfinance.gov"
```

Comment:
@OrlandoSoto is this translation okay?

Reply:
or would @doguin mind reviewing these Spanish translations?

Reply:
> or would @doguin mind reviewing these Spanish translations?

@sonnakim this one looks good to me! Search = Buscar